ui/css/perf.css

```diff
@@ -170,6 +170,7 @@ to { opacity: 0; }
   margin-bottom: 10px;
   width: 100%;
   height: 100px;
+  cursor: crosshair;
```

Comment:
@moijes12 I think we want this instead on the `#graph` container. It contains the scatter graph when you graph a series. You can try this on production or in your own local instance by graphing some sample data via the *'Add test data'* button in the UI.

There appears to be some inline styling applied to the graph container (I haven't dug around a lot in perf to see why) but it may need to be overridden to apply the cursor style.